internal/config/testdata/valid_log_config.yaml

```diff
@@ -0,0 +1,4 @@
+write:
+  create-empty-file: true
+log:
```

Comment:
lets use common file for all valid config and common testcase
https://github.com/GoogleCloudPlatform/gcsfuse/blob/tmp_sept_release/internal/config/testdata/valid_config.yaml

Reply:
done